src/KeeperIncentivesFactory.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity 0.8.19;
+
+import {BasefeeOSMDeviationCallBundler} from "mgl-keeper-incentives/Callers/BasefeeOSMDeviationCallBundler.sol";
+
+contract KeeperIncentivesFactory {
```

Comment:
The `acceptedDeviation` variable of the `BasefeeOSMDeviationCallBundler` should really be a constructor parameter.  No need to make changes in this PR, but I've made an issue for future.

https://github.com/money-god/mgl-keeper-incentives/issues/7